src/posit/connect/client.py

```diff
@@ -50,6 +50,9 @@ def connect_version(self):
     def me(self) -> User:
         url = urls.append_path(self.config.url, "v1/user")
         response = self.session.get(url)
+        body = response.json()
+        guid = body["guid"]
```

Comment:
```suggestion
        # Build the canonical URL for the User based on its GUID
        guid = body["guid"]
```

Side note: it would be nice if the API didn't require us to construct this URL, either by doing some kind of redirect to the `v1/users/guid/` location or returning it in a Location header or something.

Reply:
Agreed. Some minimal link relations in the response object would go a long way.